### ISSUE TITLE
ChaiScript Libraries

### ISSUE THREAD

**Issue by RobLoach** [RobLoach/ChaiLove#185]
Port some Lua libraries over to Chai:
- https://github.com/kikito/bump.lua

**Comment by tinglar** [comment]
Wouldn't porting bump.lua add more bloat to ChaiLove, especially since Box2D is already there? How would the option to switch between Box2D and bump.lua affect the size and performance of ChaiLove games?

My idea in dealing with those libraries was to translate the Lua(/LÖVE) libraries to ChaiScript, having those libraries be separate .chai files that developers could link to their ChaiLove game files (the same way that Player.chai is linked to main.chai in the pong example) in a case-by-case basis.

**Comment by RobLoach** [comment]
Yes, Box2D has a space in ChaiLove, but not bump.lua. We could port it directly to ChaiScript, however, outside of the C++ space. Have it as a library that you could use with:

```
love.filesystem.load("bump.chai")
```
